css/css-break/widows-008.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Fragmentation level 3 Test: 'widows' and content distribution in columns</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-break-3/#widows-orphans">
+  <link rel="match" href="reference/widows-008-ref.html">
+
+  <!--
+  Date created: November 7th 2020
+  Last modified: November 16th 2020
+  -->
+
+  <!--
+
+  Bug 1562420: Implement orphans and widows for individual columns of a multicolumn element
+  https://bugzilla.mozilla.org/show_bug.cgi?id=1562420
+
+  Bug 137367: Implement orphans and widows
+  https://bugzilla.mozilla.org/show_bug.cgi?id=137367
+
+  -->
+
+  <meta name="flags" content="">
+  <meta name="assert" content="When column boxes are filled sequentially, their content should be adjusted and fragmented in accordance with the 'widows' declaration. In the test, since the 3rd column box was going to get only 1 line box, then a class B break point is allowed and should occur before the last line box of the 2nd column box and after the next-to-last line box of the 2nd column box so that there is a minimum of 2 line boxes at the top of the 3rd column box.">
+
+  <!--
+
+    Breaking at a class B break point is allowed only if (...)
+    the number of line boxes between the break and the end
+    of the box is the value of widows or more.
+    4.4 Unforced Breaks
+    https://www.w3.org/TR/css-break-3/#unforced-breaks
+
+  -->
+
+  <style>
+  div
+    {
+      border: orange solid 4px;
+      font-size: 20px;
+      height: 104px; /* Therefore, exactly 4 line boxes */
```

Comment:
Swap this line with the next one, so that the comments appear in a more intuitive order?